doc/contributing/docs/localization/guide.rst

```diff
@@ -66,12 +66,9 @@ Term choice examples
         *   -   state machine
             -   машина состояний (конечный автомат, state machine)
             -   машина состояний
-        *   -   “write-ahead log (WAL)” at first entry, “WAL” later
```

Comment:
why?

Reply:
The examples are plain terms without descriptions. The previous version includes descriptions, which might create confusion and is also inconsistent with the previous example. Same for the punctuation.